src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -309,7 +326,13 @@ export const isSupportedThorchainSaversAssetId = (assetId: AssetId) =>
 export const isSupportedThorchainSaversChainId = (chainId: ChainId) =>
   SUPPORTED_THORCHAIN_SAVERS_CHAIN_IDS.includes(chainId)
 
-export const waitForSaversUpdate = () => setTimeoutAsync(SAVERS_UPDATE_TIME)
+export const waitForSaversUpdate = (txHash: string) =>
+  poll({
+    fn: () => getThorchainTransactionStatus(txHash),
+    validate: status => Boolean(status && status === TxStatus.Confirmed),
+    interval: 30000,
```

Comment:
We may want to make this lower and increase max attempts/make it a different interval depending on the chain family